cmd/latest_test.go

```diff
@@ -65,13 +65,46 @@ var _ = Describe("Latest", func() {
 		err := afs.Remove(core.ConfigPaths[0])
 		Expect(err).To(BeNil())
 
-		_, err = latestPipeline(afs)
+		_, err = latestPipeline(afs, false)
 		Expect(err).NotTo(BeNil())
 	})
 
 	It("fails if unable to get versions", func() {
 		// no files, means bad read for get versions
-		_, err := latestPipeline(afs)
+		_, err := latestPipeline(afs, false)
 		Expect(err).NotTo(BeNil())
 	})
+
+	makePrereleases := func(afs afero.Afero) {
+		_, err := afs.Create("chgs/not-a-version.md")
```

Comment:
I would create the files in the tests, you also don't need nearly this many files for the test cases.

Reply:
ok your call. Done